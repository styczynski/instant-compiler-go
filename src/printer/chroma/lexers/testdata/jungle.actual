project.manifest = manifest.xml

#here is a comment
base.barrelPath = barrels

fr235.barrelPath = barrels/IconLibrary.barrel;barrels/GraphLibrary.barrel
fr645m.barrelPath = barrels/M.barrel;[A/A.jungle;A/B.jungle]
round-215x215.GraphLibrary.annotations = bar

round.resourcePath = $(base.resourcePath);resource-round
DigitsPicker = [..\ScreenPickers\monkey.jungle]
base.barrelPath = $(base.barrelPath);$(DigitsPicker)
d2bravo.resourcePath = $(d2bravo.resourcePath);resources-round-218x218-ciq_1.x;resources-ciq_1.x-no_hr
d2bravo_titanium.resourcePath = $(d2bravo_titanium.resourcePath);resources-round-218x218-ciq_1.x;resources-ciq_1.x
