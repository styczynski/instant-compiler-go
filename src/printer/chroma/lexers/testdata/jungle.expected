[
  {"type":"Name","value":"project"},
  {"type":"Punctuation","value":"."},
  {"type":"Name","value":"manifest"},
  {"type":"Text","value":" "},
  {"type":"Operator","value":"="},
  {"type":"Text","value":" manifest.xml\n\n"},
  {"type":"CommentSingle","value":"#here is a comment\n"},
  {"type":"NameConstant","value":"base"},
  {"type":"Punctuation","value":"."},
  {"type":"NameBuiltin","value":"barrelPath"},
  {"type":"Text","value":" "},
  {"type":"Operator","value":"="},
  {"type":"Text","value":" barrels\n\n"},
  {"type":"Name","value":"fr235"},
  {"type":"Punctuation","value":"."},
  {"type":"NameBuiltin","value":"barrelPath"},
  {"type":"Text","value":" "},
  {"type":"Operator","value":"="},
  {"type":"Text","value":" barrels/IconLibrary.barrel"},
  {"type":"Punctuation","value":";"},
  {"type":"Text","value":"barrels/GraphLibrary.barrel\n"},
  {"type":"Name","value":"fr645m"},
  {"type":"Punctuation","value":"."},
  {"type":"NameBuiltin","value":"barrelPath"},
  {"type":"Text","value":" "},
  {"type":"Operator","value":"="},
  {"type":"Text","value":" barrels/M.barrel"},
  {"type":"Punctuation","value":";["},
  {"type":"Text","value":"A/A.jungle"},
  {"type":"Punctuation","value":";"},
  {"type":"Text","value":"A/B.jungle"},
  {"type":"Punctuation","value":"]"},
  {"type":"Text","value":"\n"},
  {"type":"NameConstant","value":"round-215x215"},
  {"type":"Punctuation","value":"."},
  {"type":"Name","value":"GraphLibrary"},
  {"type":"Punctuation","value":"."},
  {"type":"NameBuiltin","value":"annotations"},
  {"type":"Text","value":" "},
  {"type":"Operator","value":"="},
  {"type":"Text","value":" bar\n\n"},
  {"type":"NameConstant","value":"round"},
  {"type":"Punctuation","value":"."},
  {"type":"NameBuiltin","value":"resourcePath"},
  {"type":"Text","value":" "},
  {"type":"Operator","value":"="},
  {"type":"Text","value":" "},
  {"type":"Punctuation","value":"$("},
  {"type":"NameConstant","value":"base"},
  {"type":"Punctuation","value":"."},
  {"type":"NameBuiltin","value":"resourcePath"},
  {"type":"Punctuation","value":");"},
  {"type":"Text","value":"resource-round\n"},
  {"type":"Name","value":"DigitsPicker"},
  {"type":"Text","value":" "},
  {"type":"Operator","value":"="},
  {"type":"Text","value":" "},
  {"type":"Punctuation","value":"["},
  {"type":"Text","value":"..\\ScreenPickers\\monkey.jungle"},
  {"type":"Punctuation","value":"]"},
  {"type":"Text","value":"\n"},
  {"type":"NameConstant","value":"base"},
  {"type":"Punctuation","value":"."},
  {"type":"NameBuiltin","value":"barrelPath"},
  {"type":"Text","value":" "},
  {"type":"Operator","value":"="},
  {"type":"Text","value":" "},
  {"type":"Punctuation","value":"$("},
  {"type":"NameConstant","value":"base"},
  {"type":"Punctuation","value":"."},
  {"type":"NameBuiltin","value":"barrelPath"},
  {"type":"Punctuation","value":");$("},
  {"type":"Name","value":"DigitsPicker"},
  {"type":"Punctuation","value":")"},
  {"type":"Text","value":"\n"},
  {"type":"Name","value":"d2bravo"},
  {"type":"Punctuation","value":"."},
  {"type":"NameBuiltin","value":"resourcePath"},
  {"type":"Text","value":" "},
  {"type":"Operator","value":"="},
  {"type":"Text","value":" "},
  {"type":"Punctuation","value":"$("},
  {"type":"Name","value":"d2bravo"},
  {"type":"Punctuation","value":"."},
  {"type":"NameBuiltin","value":"resourcePath"},
  {"type":"Punctuation","value":");"},
  {"type":"Text","value":"resources-round-218x218-ciq_1.x"},
  {"type":"Punctuation","value":";"},
  {"type":"Text","value":"resources-ciq_1.x-no_hr\n"},
  {"type":"Name","value":"d2bravo_titanium"},
  {"type":"Punctuation","value":"."},
  {"type":"NameBuiltin","value":"resourcePath"},
  {"type":"Text","value":" "},
  {"type":"Operator","value":"="},
  {"type":"Text","value":" "},
  {"type":"Punctuation","value":"$("},
  {"type":"Name","value":"d2bravo_titanium"},
  {"type":"Punctuation","value":"."},
  {"type":"NameBuiltin","value":"resourcePath"},
  {"type":"Punctuation","value":");"},
  {"type":"Text","value":"resources-round-218x218-ciq_1.x"},
  {"type":"Punctuation","value":";"},
  {"type":"Text","value":"resources-ciq_1.x\n"}
]
